pubspec.yaml

```diff
@@ -24,6 +24,7 @@ dev_dependencies:
     sdk: flutter
   pedantic: ^1.9.0
   test: ^1.15.2
+  mocktail: ^0.3.0
```

Comment:
After applying the remarks I left in fake_converted_query_test.dart, you can safely remove the dependency to mocktail.

Reply:
Done as part of previous commit